remoteit/desktop
# [DESK-752] setting connection route

### Description


Adding routing attribute to control p2p or proxy or failover behavior

### Test plan



1. Sign in
2. Go to your device edit page
3. Click a service or add a new service
4. You should see the routing selection options
5. Save and to go the service connection page
6. You should see routing options dropdown
7. Proxy with failover allows override, p2p or proxy only will not

### Ticket



<https://remoteit.atlassian.net/browse/DESK-752>
